src/lib.rs

```diff
@@ -1666,7 +1666,7 @@ impl<'a> Serializer<'a> {
                 return Ok(());
             }
             Type::VarArgs => {
-                write!(self.w, "...")?;
+                write!(self.w, "..=")?;
```

Comment:
another string, this time for varargs output. So I think this one might actually cause test failures